components/LoginForm.tsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import styles from '../styles/Login.module.css'
+
+interface LoginFormProps {
+  onSubmit: any,
+  buttonText: string,
+}
+
+function LoginForm({onSubmit, buttonText}:LoginFormProps) {

```

Comment:
Let's be consistent. Please use anonymous functions (`() =>`) instead of `function`.

```suggestion
const LoginForm = ({onSubmit, buttonText}:LoginFormProps) => {

```

Reply:
This comment applies to all functions written with `function` keyword